<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeptuneThermodynamics</class>
 <widget class="QWidget" name="NeptuneThermodynamics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="ThermodynamicsFieldView" name="tabFieldProperties">
      <attribute name="title">
       <string>Field properties</string>
      </attribute>
     </widget>
     <widget class="ThermodynamicsSaturationView" name="tabSaturationProperties">
      <attribute name="title">
       <string>Saturation properties</string>
      </attribute>
     </widget>
     <widget class="ThermodynamicsInteractionView" name="tabInteractionProperties">
      <attribute name="title">
       <string>Interaction properties</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThermodynamicsFieldView</class>
   <extends>QWidget</extends>
   <header location="global">ThermodynamicsFieldView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThermodynamicsSaturationView</class>
   <extends>QWidget</extends>
   <header>ThermodynamicsSaturationView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThermodynamicsInteractionView</class>
   <extends>QWidget</extends>
   <header>ThermodynamicsInteractionView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
